.storybook/config.js

```diff
@@ -7,10 +7,5 @@ const req = require.context("../stories", true, /\.stories\.tsx$/);
 function loadStories() {
   req.keys().forEach(req);
 }
- 
-const ToastDecorator = storyFn => (
-  <ToastContextProvider>{storyFn()}</ToastContextProvider>
-)
```

Comment:
Is there any special reason to move `stories`?

Reply:
We don't need to add this decorator. I included multiple providers directly within the stories file